seatunnel-e2e/seatunnel-connector-v2-e2e/connector-pulsar-e2e/src/test/resources/source_json_to_console.conf

```diff
@@ -0,0 +1,97 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  execution.parallelism = 1
+  job.mode = "BATCH"
+  # You can set spark configuration here
+  spark.app.name = "SeaTunnel"
+  spark.executor.instances = 1
+  spark.executor.cores = 1
+  spark.executor.memory = "1g"
+  spark.master = local
+}
+
+source {
+  Pulsar {
+    topic = "json_topic_source"
+    subscription.name = "seatunnel"
+    client.service-url = "pulsar://pulsar-e2e:6650"
+    admin.service-url = "http://pulsar-e2e:8080"
+    result_table_name = "pulsar_table"
+    format_error_handle_way = skip
+    cursor.startup.mode = "EARLIEST"
+    cursor.stop.mode = "LATEST"
+    schema = {
+      fields {
+        id = bigint
+        c_map = "map<string, smallint>"
+        c_array = "array<tinyint>"
+        c_string = string
+        c_boolean = boolean
+        c_tinyint = tinyint
+        c_smallint = smallint
+        c_int = int
+        c_bigint = bigint
+        c_float = float
+        c_double = double
+        c_decimal = "decimal(2, 1)"
+        c_bytes = bytes
+        c_date = date
+        c_timestamp = timestamp
+      }
+    }
+  }
+}
+
+transform {
+}
+
+sink {
+  Console {
+    source_table_name = "pulsar_table"
+  }
+  Assert {
+    source_table_name = "pulsar_table"
+    rules =
```

Comment:
The best way to test this case is add a Pulsar Sink or LocalFileSink and check every row write in the Pulsar or local file.

Reply:
I agree with you, in my opinion `Pulsar Sink` could be perfected by creating a new PR alone. The current PR is more about solving the related problems.

Reply:
> I agree with you, in my opinion `Pulsar Sink` could be perfected by creating a new PR alone. The current PR is more about solving the related problems.

Can you create an issue about this?

Reply:
#4737